check-code-compliance/action.yaml

```diff
@@ -153,6 +153,7 @@ runs:
               Public Domain; \
               WTFPL; \
               CC-BY-4.0; \
+              Unicode-DFS-2016; \
```

Comment:
@csernalabics do you have 5 cents?

Reply:
> YOU MUST Forward Copyright notices In Files
> YOU MUST Forward License text In Files
> YOU MUST Provide Copyright notices In Documentation
> YOU MUST Provide License text In Documentation
> YOU MUST NOT Promote

Aus meiner Sicht sollte das passen, da Open Source und kein Copyleft; s. auch hier: https://blueoakcouncil.org/list